src/bottomSheet.js

```diff
@@ -43,225 +42,130 @@ function BottomSheet(props) {
     velocityThreshold = 0.9,
     distanceThreshold = 150,
     closeOnOverlayClick = true,
-    animateOnDrag = {},
     onDragStart = () => {},
     onDragEnd = () => {},
+    scrollableSheet = true,
   } = props;
+
+  let { content = "", draggableArea = `` } = props;
+  function getSnapPointAnimation() {
+    return `spring(1, 95, 25, 13)`;
+  }
+
   let lastSetSnapPoint;
   content =
     typeof content !== "string"
-      ? promise.resolve(content).then((value) => {
-          value;
-        })
+      ? Promise.resolve(content).then(value => value)
       : content;
-  let targetid = trigger
+  const targetid = trigger
     ? document
         ?.querySelector(`#${trigger}`)
         ?.getAttribute("data-bottomsheet-id")
     : "";
   let targetBottomSheet = targetid
     ? document?.querySelector(`#${targetid}`)
     : "";
+  function getCurrentSnapPoint(newBottomSheet) {
```

Comment:
```suggestion
    : "";
    
  function getCurrentSnapPoint(newBottomSheet) {
```

For better readability, alwasy leave a line between the function declaration or any piece of important code.

They will get removed in build anyways.